rasa/shared/core/generator.py

```diff
@@ -217,6 +217,26 @@ def update(self, event: Event, skip_states: bool = False) -> None:
 
             self._append_current_state()
 
+    def fingerprint(self) -> Text:
+        """Returns a unique hash for the tracker which is stable across python runs.
+
+        Returns:
+            fingerprint of the tracker
+        """
+        data = {"sender_id": self.sender_id}
+
+        if self.domain:
+            data.update(self.domain.as_dict())
```

Comment:
We can just add the whole `domain` as it's fingerprint-able right?

Reply:
Would adding to the `data` dict this key-value pair work: `{'domain': self.domain.fingerprint()}`?

Reply:
You can just do `{'domain': self.domain}`. As `deep_container_fingerprint` can now handle `.fingerprint`